src/tests/dbus-tests/test_nvme.py

```diff
@@ -0,0 +1,344 @@
+import os
+import stat
+import re
+import tempfile
+import time
+import sys
+import json
+import uuid
+import dbus
+import unittest
+import udiskstestcase
+
+from distutils.spawn import find_executable
```

Comment:
Please use `shutil.which` instead of of `find_executable`, the distutils mode is deprecated and will be removed in Python 3.12.

Reply:
Fixed.